browser/src/layer/tile/CanvasTileWorker.js

```diff
@@ -0,0 +1,116 @@
+/* -*- js-indent-level: 8; fill-column: 100 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/* eslint-disable no-inner-declarations */
+/* eslint no-unused-vars: ["warn", { "argsIgnorePattern": "^_" }] */
+/* global importScripts Uint8Array */
+
+if ('undefined' === typeof window) {
+	self.L = {};
+
+	importScripts('../../../node_modules/fzstd/umd/index.js');
+	importScripts('CanvasTileUtils.js');
```

Comment:
I don't know that this is an ok thing to do, or what the expected way of doing this is - I would certainly appreciate some help here. How do sources get included in the main non-worker JS files?

Reply:
@eszkadev any ideas on this importScripts thing. Presumably the single ./browser/dist/bundle.js case poses an issue with that?

Reply:
We don't use any standard bundler, but Makefile. Just add entry in Makefile.am and it will be included in our bundle.